ci/teamcity/comment_on_pr.py

```diff
@@ -50,11 +51,15 @@
         )
         for i in content.split("+ python3 -m pytest ")
     )
+    tests_failed = tests_failed or ("FAILURES" in full_comment)
```

Comment:
why not like this?

```suggestion
    tests_failed = "FAILURES" in full_comment
```

Reply:
Because it is a loop over `pytest_outputs`.

Reply:
I see